src/main/resources/com/adaptavist/tm4j/jenkins/extensions/configuration/Tm4jGlobalConfiguration/config.jelly

```diff
@@ -2,30 +2,37 @@
 <j:jelly xmlns:j="jelly:core"
          xmlns:f="/lib/form">
     <f:section title="Test Management for Jira">
-
-        <f:entry title="Jira server(s)">
-            <f:repeatable var="tm4jInstancesList" name="jiraInstances" items="${instance.jiraInstances}"
+        <f:entry title="Jira instance(s)">
+            <f:repeatable varStatus="status" var="tm4jInstancesList" name="jiraInstances" items="${instance.jiraInstances}"
                           noAddButton="false" minimum="0">
                 <table>
```

Comment:
I think would be nice to add a link to the documentation of each product (Server/Cloud) somewhere here. Please check out with our POs what they think? May be a separate PR/task though...

Reply:
Each help section has the link to the Wiki page - I wouldn't add another link in there.

Reply:
Oh, awesome, I didn't know that! I suppose by wiki page you mean doco...

Reply:
No, I mean the Jenkins wiki